dashboard/src/main/java/com/google/cloud/tools/opensource/cloudbomdashboard/DashboardMain.java

```diff
@@ -178,7 +178,7 @@ private static ArtifactCache buildCache(Bom bom) {
     return loadArtifactInfo(managedDependencies);
   }
 
-  private static boolean report(Bom bom) {
+  private static boolean report(Bom bom) throws IOException {
```

Comment:
Consider passing in an `OutputStream` or `OutputStreamWriter` and configure to either write to a file or standard out in the plugin setup.